lib/veteran_enrollment_system/associations/configuration.rb

```diff
@@ -10,7 +10,7 @@ def self.api_key_path
       end
 
       def base_path
-        "#{Settings.veteran_enrollment_system.host}/ves-associate-gateway-svc/associations/person/"
+        "#{super}ves-associate-gateway-svc/associations/person/"
```

Comment:
I don't see this pattern being used anywhere else in the codebase. I like the idea of being able to search for `Settings.veteran_enrollment_system` and see where all it's used. `super` makes it a little less clear. But, if there's a good reason to do it, that's fine.

Reply:
I thought it would be a little more efficient this way, since the service class is a subclass of the `base_configuration`, which houses the `base_path` for all of the subclasses. However, if it's not a common pattern, I can change it back. Thanks @rmtolmach!

Reply:
Awesome, thanks! Looks like you have a linting failure now, though 🫠

Reply:
Haha, of course 🫠 We're good now! 😄